### PR TITLE
[BD-6] Add codejail python 3.8 test

### PR DESCRIPTION
**Pull request by morenol**
Add jenkins python3.8 test and remove python2.7 tests for codejail

## Reviewers
- [ ] @awais786 
- [ ] @ericfab179

### CODE REVIEW

File: testingPipelines/jobs/generalPRPipeline.groovy [hunk 1]
```diff
@@ -54,9 +54,26 @@ Map codejail35JobConfig = [
...
+    jenkinsFileName: 'Jenkinsfile',
+    environmentVariables: [
+        'TOX_ENV': 'py38',
+        'PYTHON_VERSION': '3.8'
+    ]
+]
+
+
 List jobConfigs = [
     codejail27JobConfig,
```
Comment (morenol) [inline testingPipelines/jobs/generalPRPipeline.groovy:74]: Should I remove the python 2.7 tests?

**Comment by edx-webhook** [comment]
Thanks for the pull request, @morenol! I've created [OSPR-4719](https://openedx.atlassian.net/browse/OSPR-4719) to keep track of it in JIRA, where we prioritize reviews.  Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).

Please let us know once your PR is ready for our review and all tests are green.

File: testingPipelines/jobs/generalPRPipeline.groovy (hunk 1 above)
Comment (morenol) [reply testingPipelines/jobs/generalPRPipeline.groovy]: removed

**Comment by awais786** [comment]
@morenol this PR also runs one check. But last merged PR shows 2 check.

File: testingPipelines/jobs/generalPRPipeline.groovy [hunk 2]
```diff
@@ -24,39 +24,40 @@ catch (any) {
...
+Map codejail35JobConfig = [
     open: true,
-    jobName: 'codejail-python-2-pr',
+    jobName: 'codejail-python-35-pr',
     repoName: 'codejail',
-    context: 'jenkins/python2',
+    context: 'jenkins/python35',
     onlyTriggerPhrase: false,
-    triggerPhrase: /.*jenkins\W+test\W+python\W+2.*/,
+    triggerPhrase: /.*jenkins\W+run\W+python.*/,
```
Comment (morenol) [inline testingPipelines/jobs/generalPRPipeline.groovy:33]: I changed the regex to be more similar to the one used in edx-platform

**Pull request closed without merging**